core/src/test/scala/integration/kafka/api/EndToEndAuthorizationTest.scala

```diff
@@ -64,15 +63,10 @@ import scala.jdk.CollectionConverters._
   * SaslTestHarness here directly because it extends QuorumTestHarness, and we
   * would end up with QuorumTestHarness twice.
   */
+@Timeout(60)
```

Comment:
why do we need Timeout?

Reply:
In my testing which included the added KRAFT support, some tests failed without increasing the default timeout. @cmccabe suggested I put the timeout on the class.